action.yml

```diff
@@ -14,6 +14,14 @@ runs:
     - name: Verify NuGet package signatures
       run: Get-ChildItem -Recurse -File nugets -Filter *.nupkg -ErrorAction:Ignore | foreach {dotnet nuget verify $_.FullName}
       shell: pwsh
+    - name: Verify assets signatures
+      if: runner.os == 'Windows'
+      run:  Get-ChildItem -Recurse -File assets -Filter *.exe -ErrorAction:Ignore | foreach {&"C:\Program Files (x86)\Windows Kits\10\bin\10.0.19041.0\x86\signtool.exe" verify /pa $_.FullName}
```

Comment:
Question: Do we know under what conditions the path to signtool.exe will change? Or is there any way we can discover the path rather than hardcoding it?

From the supported software pages for [win-2022](https://github.com/actions/runner-images/blob/main/images/windows/Windows2022-Readme.md?plain=1#L469-L471) and [win-2019](https://github.com/actions/runner-images/blob/main/images/windows/Windows2019-Readme.md?plain=1#L414-L420) it does look like the 2 images currently share 19041 in common but both also have 20348 but then there's the Win11 SDK as well?

Reply:
@DavidBoike we updated the action after this PR to be less hardcoded
https://github.com/Particular/push-octopus-package-action/blob/main/action.yml#L17

Reply:
Cool, very nice! Sorry for the distraction.

Reply:
No issues, I appreciate you looking at it.